hazelcast/src/main/java/com/hazelcast/internal/dynamicconfig/DynamicConfigYamlGenerator.java

```diff
@@ -660,6 +662,28 @@ public static void pnCounterYamlGenerator(Map<String, Object> parent, Config con
         parent.put("pn-counter", child);
     }
 
+    public static void externalDataStoreYamlGenerator(Map<String, Object> parent, Config config) {
```

Comment:
Minor: Maybe we can do this(and DynamicConfigYamlGeneratorTest) later when you send the dynamic configuration PR?

Reply:
I'll create the PR right after this one being merged, if you don't mind I would leave it as it is

Reply:
Ok, sounds good to me